README.md

```diff
@@ -3,55 +3,85 @@
+For the latter, a financial engineer may want to test their strategies against thousands of market conditions, contract settings, shocks, and autonomous or random or even AI agents all while making sure their strategy isn't vulnerable to bytecode-level exploits.
+
+To configure such a rich simulation environment on a test or local network is also possible with Arbiter by a change in choice of middleware. 
+The most efficient choice for getting robust, yet quick, simulations would bypass any networking and use a low level language's implementation of the EVM. 
+Furthermore, we can gain control over the EVM worldstate by working directly on `revm`.
+We would like the user to have a choice in how they want to simulate their contracts and Arbiter provides that choice.
+
+### Sim Driven Development and Strategization 
+
+Test driven development is a popular engineering practice to write tests first, which fail, and implement logic to get the test to eventually pass. With simulation driven development, it's possible to build "tests" that can only pass if the *incentives* actually work. For example, a sim driven test might be `is_loan_liquidated`, and a simulation must be made for a liquidator agent to do the liquidation. This approach significantly improves the testing of economic systems and other mechanism designs, which is important in the world of networks that are mostly incentive driven.
+
+The same goes with developing strategies that one would like to deploy on a live Ethereum network. One can use Arbiter to simulate their strategy with an intended goal and see if it actually works. This is especially important in the world of DeFi where strategies are often a mix of on and offchain and are susceptible to exploits.
 
 ## Installation
 
-### Build from source
+To install Arbiter, you will need to have Rust installed on your machine. You can install Rust by following the instructions [here](https://www.rust-lang.org/tools/install). Once you have Rust installed, you can install Arbiter by running the following commands:
 
 ```bash
 git clone https://github.com/primitivefinance/arbiter.git
-cd arbiter
-cargo install --path .
+cargo install --path ./arbiter
 ```
+This will install the Arbiter binary on your machine. You can then run `arbiter --help` to see that Arbiter was installed properly as well as see the help menu.
 
-## CLI 
+## Command Line Interface 
+
+The Arbiter binary provides a CLI for creating new projects much like [Foundry](https://github.com/foundry-rs/foundry), which Arbiter aims to work alongside with. To create a new project, you can run:
 
 ```bash
 arbiter init your-project-name
 cd your-project-name
-arbiter bind
+```
+
+This initializes a new Arbiter project with a template. The template is executable at this point and you can run it by running:
+```bash
 cargo run
```

Comment:
This wont run if you don't create template bindings first.

Reply:
Oh yeah I suppose that's true.

Here's my suggestion, can we run `bind` upon the first `init` so that the template can be immediately run?